### PR TITLE
Feature/pip install instructions

### PR DESCRIPTION
**Pull request by fisher60**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,49 +1,72 @@
...
 ## Installation
 This is simple with pip. Just run the following in your command line or terminal (available soon):
+
 ```
 pip install bestpy
 ```
 
-You can also use your magic powers to get the module from source code. Though we won't be guiding you with that hokus-pokus here. Ask a friend if your powers aren't strong enough.
+You can also use your magic powers to get the module from source code. 
+By using:
```
Comment (gustavwilliam) [inline README.md]:
```suggestion
You can also use your magic powers to get the module from source code. Use the following:
```
I'm not a huge fan of starting a sentence with "By using"

File: README.md [hunk 2]
```diff
@@ -1,49 +1,72 @@
...
 ```
 pip install bestpy
 ```
 
-You can also use your magic powers to get the module from source code. Though we won't be guiding you with that hokus-pokus here. Ask a friend if your powers aren't strong enough.
+You can also use your magic powers to get the module from source code. 
+By using: 
+
+```
+pip install git+https://github.com/gustavwilliam/bestpy.git@main
```
Comment (gustavwilliam) [inline README.md:24]: This worked for me after restarting the terminal. Maybe we could add a note about that restarting after running the command could help.

**Review by gustavwilliam — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -18,12 +18,13 @@ This is simple with pip. Just run the following in your command line or terminal
...
 
-You can also use your magic powers to get the module from source code. 
-By using: 
+You can also use your magic powers to get the module from source code with the following:
 
 ```
 pip install git+https://github.com/gustavwilliam/bestpy.git@main
-```
+
+# Note: you will likely need to restart your terminal before using the module
```
Comment (gustavwilliam) [inline README.md]: Maybe we could add this outside of the code block? It seems a bit off to have it inside.

**Review by gustavwilliam — CHANGES_REQUESTED** [review]
Looks almost good to me

File: README.md [hunk 4]
```diff
@@ -1,49 +1,73 @@
...
+
 ```
 pip install bestpy
 ```
 
-You can also use your magic powers to get the module from source code. Though we won't be guiding you with that hokus-pokus here. Ask a friend if your powers aren't strong enough.
+You can also use your magic powers to get the module from source code with the following:
+
+```
+pip install git+https://github.com/gustavwilliam/bestpy.git@main
```
Comment (Akarys42) [inline README.md:24]: Maybe we could setup a `bestpy-nightly` release, I’m sure this would be really useful.

Comment (gustavwilliam) [reply README.md:24]: I haven't heard about 'nightly'. If you think it's something we should add/use/implement, feel free to open an issue about it. I'd love to hear more regarding your thoughts.

**Review by gustavwilliam — APPROVED** [review]
LGTM. Let's do it.

**Pull request merged**